Week1/README.md

```diff
@@ -21,187 +16,13 @@ These are the topics for week 1:
    - What does a database do for our apps?
    - How to connect with Node.js -->
 
-## 0. Video Lectures
+## Week goals
 
-Your teachers Igor and Unmesh have made video lectures for this week's material. You can find them here: [Videos 1 - 8](https://www.youtube.com/playlist?list=PLVYDhqbgYpYVq-rHFB1x2-P7esnWotCB7)
+This week we want to introduce you to the world of databases! We will start with a general introduction to what databases are, going over some terms and how to start thinking about your application's data. Have a look for the introduction [here](https://hackyourfuture.github.io/study/#/databases/README).
 
-<a href="https://www.youtube.com/playlist?list=PLVYDhqbgYpYVq-rHFB1x2-P7esnWotCB7" target="_blank"><img src="../assets/playlist-thumbnail.png" width="600" height="350" alt="HYF Video" /></a>
+In this module we will be covering both the relational type of databases (SQL) as well as document based databases (NoSQL). For now you do not need to know the difference, that will become apparent when we have covered both. We will start by going into relational database by learning about SQL, which is the language used when interacting with most relational databases. Have a look [here](https://hackyourfuture.github.io/study/#/databases/SQL).
 
-## 1. What is an information (system)?
```

Comment:
Removing all these contents make `LESSONPLAN` and `README` out of sync

Reply:
Good point, resynced them by providing subtopics where needed